build.sbt

```diff
@@ -306,7 +300,7 @@ addCommandsAlias(
     "+coverage",
     "+test",
     "+coverageReport",
-    "+mimaReportBinaryIssues",
+    //"+mimaReportBinaryIssues",
```

Comment:
I'd suggest to make a commit to the series-branch where we set `mimaPreviousArtificats` to empty instead.